docs/GettingStarted.md

```diff
@@ -98,6 +98,14 @@ Node.js comes with npm, which lets you install the React Native command line int
 npm install -g react-native-cli
 ```
 
+Ensure your version of npm is greater than 3:
+```
+npm --version
+```
+If it is not you can update npm:
```

Comment:
In general there are a lot of reasons that people can't update npm, so I'm concerned about adding npm 3+ as a requirement for React Native. Could this be backward-compatible instead of a breaking change somehow?